File: fixtures/scale/kotlin/large/src/cryptobench/registry/StreamRegistry.kt
package cryptobench.registry

import cryptobench.core.CipherSuite
import cryptobench.ciphers.xorcipher.XorCipherSuite
import cryptobench.ciphers.rc4.Rc4Suite
import cryptobench.ciphers.xorshift.XorShiftSuite
import cryptobench.ciphers.lcg.LcgSuite

object StreamRegistry {
    fun suites(): List<CipherSuite> = listOf(
        XorCipherSuite(),
        Rc4Suite(),
        XorShiftSuite(),
        LcgSuite(),
    )
}

File: fixtures/scale/dart/large/lib/registry/stream_registry.dart
import '../core/cipher_suite.dart';
import '../ciphers/xorcipher/xorcipher_suite.dart';
import '../ciphers/rc4/rc4_suite.dart';
import '../ciphers/xorshift/xorshift_suite.dart';
import '../ciphers/lcg/lcg_suite.dart';

List<CipherSuite> streamSuites() => [
  XorCipherSuite(),
  Rc4Suite(),
  XorShiftSuite(),
  LcgSuite(),
];
